### PR TITLE
feat: docs and test refactoring for maxLength validation

### PR DESCRIPTION
**Pull request by ariskemper**

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you! Can you try to fix the CI problems?

**Comment by ariskemper** [comment]
@fabian-hiller could you review and merge?

File: library/src/validations/maxLength/maxLength.test.ts [hunk 1]
```diff
@@ -2,25 +2,55 @@ import { describe, expect, test } from 'vitest';
...
-    expect(validate._parse(value3).output).toBe(value3);
-
-    expect(validate._parse('1234').issues).toBeTruthy();
-    expect(validate._parse('123456789').issues).toBeTruthy();
-    expect(validate._parse([1, 2, 3, 4]).issues).toBeTruthy();
-    expect(validate._parse([1, 2, 3, 4, 5, 6, 7]).issues).toBeTruthy();
+  describe('should pass only valid lengths', () => {
+    test('empty string has valid length', () => {
+      const value1 = '';
+      expect(maxLength(3)._parse(value1).output).toBe(value1);
```
Comment (fabian-hiller) [inline library/src/validations/maxLength/maxLength.test.ts:8]: Change you change that to how we implemented it at `email` with `.toEqual({ output: value })`? Same for the rest of the tests.

**Comment by fabian-hiller** [comment]
I am closing this PR as it has been added recently. Sorry for not building on top of this PR.

**Pull request closed without merging**